managers/models.py

```diff
@@ -43,6 +46,7 @@ def __str__(self) -> str:
 def update_user_type(sender: Project, action: str, pk_set: Set, **kwargs: Any) -> None:
     assert sender == Project.managers.through
     project = kwargs["instance"]
+    logger.debug(f"{action}")
```

Comment:
This says nothing actually, please add `sender.pk` and `pk_set` too and make it more descriptive.

Reply:
Done